demo/site/src/blocks/TextLinkBlock.tsx

```diff
@@ -7,6 +7,16 @@ import { LinkBlock } from "./LinkBlock";
 
 export const TextLinkBlock = withPreview(
     ({ data: { link, text } }: PropsWithData<DemoTextLinkBlockData>) => {
+        if (link.block && link.block.type === "damFileDownload" && "file" in link.block.props && link.block.props.file) {
+            return (
+                <Link data={link}>
+                    <>
+                        {text} ({Math.round(link.block.props.file.size / 1024)} KB)
+                    </>
+                </Link>
+            );
+        }
```

Comment:
This isn't great, but I have no idea how to make it any better. Also, it will always be in the application code, so I guess it's fine.